test/cli/fixtures/fails/concurrent-suite-deadlock.test.ts

```diff
@@ -0,0 +1,41 @@
+import { describe, test, vi } from 'vitest'
+import { createDefer } from 'vitest/dist/utils.js'
```

Comment:
Why is it importing from dist 🤔

Reply:
Hmm, I didn't notice but it seems typescript got that one from autocomplete.
I installed `@vitest/utils` and it looks okay now.